drivers/SmartThings/matter-switch/fingerprints.yml

```diff
@@ -138,7 +138,7 @@ matterGeneric:
     deviceLabel: Matter Color Temperature Light
     deviceTypes:
       - id: 0x010C # Color Temperature Light
-    deviceProfileName: light-level-colorTemperature
+    deviceProfileName: light-color-level
```

Comment:
We would probably want to do this for all fingerprints involving device type `0x010C`

Reply:
I'm not sure if this change should be needed. The Color Control cluster is used for both setting the color and the color temperature. 0x010C is the Color Temperature Light device type so I would expect it to only use the Color Control cluster for setting the color temperature. A device that supports setting the color and color temperature would likely use device type 0x010D which is the Extended Color Light.